### ISSUE TITLE
Narrator is announcing as suggestions available when the keyboard focus is on ‘Previous/Next’ month buttons.

### ISSUE THREAD

**Issue by margaret-schmit** [microsoft/fluentui#15278]
### Environment Information

Windows OS Version: 2004 (Build 20161.1000)
Edge Dev Version: Version 85.0.552.1 (Official build) dev (64-bit)
NVDA Version: 2020.1

Prerequisite:
URL: https://project.microsoft.com

### Describe the issue:

 1. Navigate to a project containing tasks from Home page.

2. Now navigate to ‘Start column’ in a task and press Alt+down arrow to open calendar.

3. Now navigate to ‘Next/Previous’ month button and listen to the narrator announcement.

#### Actual behavior:
When the keyboard focus is on ‘Next/Previous Month’ buttons narrator is announcing incorrect information as “Suggestions available”.

#### Expected behavior:
When the keyboard focus is on ‘Next/Previous Month’ buttons narrator shouldn’t announce incorrect information as “Suggestions available”.

### Documentation describing expected behavior

There should not be any information that a user cannot understand through the narrator. The narrator is announcing the wrong use for this button, therefore the user is missing information about project

**Comment by miroslavstastny** [comment]
Tested in Edge 85.0.564.63, NVDA 2020.2 - NVDA does not narrate "Suggestions available":
![image](https://user-images.githubusercontent.com/9615899/94707859-36fad600-0344-11eb-8a98-ed824f1cf3ec.png)
@margaret-schmit In order for us to be able to help you, could you please provide us with a codepen example demonstrating the issue in isolation?
